utils.py

```diff
@@ -0,0 +1,213 @@
+DEFAULT_PER_FEATURE_LIMIT = 100
+DEFAULT_QUERY_TIMEOUT = 120000
+LAST_X_MINUTES = 10
+
+#Schemas
+schemas = {
+    'query_sensors':{
+    'limit':DEFAULT_LIMIT,
+    'offset':DEFAULT_OFFSET,
+    'sortDirection':DEFAULT_SORT,
+    'filters':[]
+    },
+    'simple_query_generic_attributes':{
+    'totalResultLimit': 200,
+    'perGroupLimit': 100,
+    'perFeatureLimit': 100,
+    'templateContext': 'SPECIFIC',
+    'queryTimeout': 120000
+    },
+    'query_user':{
+        'queryPath':[
+            {
+                'requestedType': 'User',
+                'filters': [],
+                'isResult': True
+            }
+        ],
+        'customFields':[
+                'domain',
+                'ownerMachine',
+                'ownerOrganization.name',
+                'ownerOrganization',
+                'isLocalSystem',
+                'elementDisplayName'
+        ]
+    },
+    'query_file': {
+        'queryPath': [
+            {
+                'requestedType': 'File',
+                'filters': [],
+                'isResult': True
+            }
+        ],
+        'customFields': [
+            'md5String',
+            'ownerUser',
+            'ownerMachine',
+            'sha1String',
+            'signatureVerified','avRemediationStatus',
+            'maliciousClassificationType',
+            'createdTime',
+            'modifiedTime',
+            'size',
+            'correctedPath',
+            'productName',
+            'productVersion',
+            'companyName',
+            'internalName',
+            'elementDisplayName',
+            'isSigned'
+        ]
+    },
+    'query_process':{
+            "queryPath": [
+            {
+                "requestedType": "Process",
+                "filters": [],
+                "isResult": True
+            }
+            ],
+            "customFields": [
+                "elementDisplayName",
+                "creationTime",
+                "endTime",
+                "commandLine",
+                "productType",
+                "children",
+                "parentProcess",
+                "ownerMachine",
+                "calculatedUser",
+                "imageFile",
+                "knownMalwareSuspicion",
+                "hasListeningConnection",
+                "scanningProcessSuspicion",
+                "tid",
+                "iconBase64",
+                "ransomwareAutoRemediationSuspended",
+                "executionPrevented",
+                "isWhiteListClassification",
+                "matchedWhiteListRuleIds"
+            ]
+        },
+    'white_blacklist':[
+                {
+                    "keys": [],
+                    "maliciousType": "blacklist",
+                    "prevent": False,
+                    "remove": False
+                }
+        ]
+}
+
+#Utils
+def sensor_payload_builder(params):
+    '''
+    Build POST payload for sensors calls
+    '''
+    payload = copy.deepcopy(schemas['query_sensors'])
```

Comment:
what is the default page size if the user provides no value or unsets the page size? does the code have some default to avoid pulling huge data accindentally?